WooCommerce/UITestsFoundation/Screens/Orders/PaymentMethodsScreen.swift

```diff
@@ -35,6 +35,12 @@ public final class PaymentMethodsScreen: ScreenObject {
     @discardableResult
     public func goBackToOrderScreen() throws -> SingleOrderScreen {
         dismissButton.tap()
+
+        let orderDetailTableView = app.tables["order-details-table-view"]
+        // On smaller screen, the Summary area might not be visible anymore. We need to bring it back so when returning
+        // back from Payment Details, it can be detected.
+        orderDetailTableView.swipeDown()
+
```

Comment:
This code was moved from `goBackToOrdersScreen` method in `SingleOrderScreen` to here, it was added in https://github.com/woocommerce/woocommerce-ios/pull/14335, but from my testing (and my understanding), the logic should be executed before that, because we'll need the order details's summary view to be visible before getting to `SingleOrderScreen`, without this change, the UI tests failed constantly on the iPhone 8 Plus.